File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
    const result = new Array(arr1.length);
    
    for (let i = 0; i < arr1.length; i++) {
        result[i] = new Array(arr2[0].length);
    }
    
    for (let i = 0; i < arr1.length; i++) {
        for (let j = 0; j < arr2[0].length; j++) {
            let sum = 0;
            for (let k = 0; k < arr1[0].length; k++) {
                sum += arr1[i][k] * arr2[k][j];
            }
            result[i][j] = sum;
        }
    }
    return result;
}

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    result = list([0 for _ in range(len(arr2[0]))] for _ in range(len(arr1)))
    
    for i in range(len(arr1)):
        for j in range(len(arr2[0])):
            temp = 0
            for k in range(len(arr1[i])):
                temp += arr1[i][k] * arr2[k][j]
            result[i][j] = temp
    
    return result